app/helpers/blacklight/layout_helper_behavior.rb

```diff
@@ -0,0 +1,20 @@
+# Methods added to this helper will be available to all templates in the hosting
+# application
+module Blacklight
+  # A module for useful methods used in layout configuration
+  module LayoutHelperBehavior
+    ##
+    # Classes to be added to a document's show content div
```

Comment:
It might be good to separate the column sizes from the semantic classes.. Or, I know we use the same column sizes in `catalog/index.html.erb` and we should probably be consistent.